core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -394,8 +394,11 @@ export class ContributionsAndReview
     if (suggestion.suggestion_type === this.SUGGESTION_TYPE_TRANSLATE) {
       const suggestionIdToContribution = {};
       for (let suggestionId in this.contributions) {
-        const contribution = this.contributions[suggestionId];
-        suggestionIdToContribution[suggestionId] = contribution;
+        if (this.contributions[suggestionId].suggestion.suggestion_type ===
+            this.SUGGESTION_TYPE_TRANSLATE) {
+          const contribution = this.contributions[suggestionId];
+          suggestionIdToContribution[suggestionId] = contribution;
+        }
```

Comment:
Rather than having to filter like this, I would think we want to make sure `this.contributions` is correctly cleared when the user switches tabs.

Reply:
Was this changed?

Reply:
There was a merge issue with these files, I think I've resolved it now.